### DISCUSSION TITLE
State of CouchDB & ecosystem?

### DISCUSSION THREAD

**Discussion by GormanFletcher** [apache/couchdb#3408] [General]
I'm evaluating implementation technologies for a product I'm working on, and CouchDB looks great for my technical needs. Easy federation, speaks HTTP, offline support, and users sharing data works exactly like I need.

But, in the business sense, I'm struggling to answer the question, "three-to-five years from now, will I be comfortable with my product being dependent on CouchDB?". I don't know how to interpret CouchDB's pace of Github activity, PouchDB's seeming-dormancy, or the degree of drop-off of CouchDB online community activity since the early-2010's NoSQL craze faded. And a lot of the online activity I do find appears closer to "trying it out" than business-critical usage of CouchDB.

What is the CouchDB project's outlook for the years to come?

**Comment by wohali** [comment]
I can't speak to all of this, but:

CouchDB 3.x is stable and will be maintained for years to come by a core, dedicated group of volunteers.

CouchDB 4.x is under active development, mainly by our community members at IBM. This is expected to modernize the clustering code by relying on FoundationDB, technology built by Apple and intended to underpin other database solutions (it is not, itself, a suitable database.)

You can read more about this here: https://www.ibm.com/cloud/blog/database-deep-dives-couchdb
